metis-authentication/metis-authentication-rest/src/main/java/eu/europeana/metis/authentication/rest/controller/AuthenticationController.java

```diff
@@ -208,15 +207,13 @@ public void updateUserToMakeAdmin(@RequestHeader("Authorization") String authori
     if (emailParameter == null || StringUtils.isBlank(emailParameter.getEmail())) {
       throw new BadContentException("userEmailToMakeAdmin is empty");
     }
-    String accessToken = authenticationService
-        .validateAuthorizationHeaderWithAccessToken(authorization);
+    String accessToken = authenticationService.validateAuthorizationHeaderWithAccessToken(authorization);
```

Comment:
suggestion to use `final` here.

Reply:
Done. I guess additional suggestion, since I did not add/change the code, just put it in one line.